installing/installing_on_prem_assisted/installing-on-prem-assisted.adoc

```diff
@@ -6,7 +6,12 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports both x86-64 and AArch64 CPU architectures.
+You can install {product-title} on on-premise hardware or on-premise VMs using the {ai-full}. Installing {product-title} using the {ai-full} supports x86_64, AArch64, ppc64le, and s390x CPU architectures.
+
+[NOTE]
+====
+Currently installing {product-title} on {ibmzProductName} (s390x) is only supported with RHEL KVM installations.
```

Comment:
You can probably remove the _Currently_. 

s/only supported/supported only

Reply:
@mburke5678 thanks.

Updated as : 
```
Installing {product-title} on {ibmzProductName} (s390x) is supported only with RHEL KVM installations.
```